server/app/services/export/QuestionJsonPresenter.java

```diff
@@ -139,7 +142,7 @@ class FileUploadJsonPresenter implements QuestionJsonPresenter<FileUploadQuestio
     @Override
     public ImmutableMap<Path, Optional<String>> getJsonEntries(FileUploadQuestion question) {
       return ImmutableMap.of(
-          question.getApplicantQuestion().getContextualizedPath().join(Scalar.FILE_KEY),
+          question.getFileKeyPath(),
```

Comment:
Where available, we use the built in path generator on the question instead of building it ourselves here.